analytics_data_api/v0/views/learners.py

```diff
@@ -168,7 +181,7 @@ def _validate_query_params(self):
 
     def get_queryset(self):
         """
-        Fetches the user list from elasticsearch.  Note that an
+        Fetches the user list from elasticsearch. Note that an
```

Comment:
I see. If it's trouble, then don't worry about it. @singingwolfboy - maybe we can chat about this plugin later?